src/components/RecipeCard.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import { TimeIcon } from '@chakra-ui/icons';
+import { Text, Flex, Spacer } from '@chakra-ui/react';
+import Img from 'gatsby-image';
+import { graphql } from 'gatsby';
+
+interface Props {
+  data: GatsbyTypes.RecipeCardFragment;
+}
+
+function RecipeCard({ data }: Props): JSX.Element {
+  return (
+    <div style={{ marginRight: '10%', marginBottom: '10%' }}>
+      <Flex>
+        <h3
+          style={{
+            fontWeight: 850,
+            float: 'left',
+            fontSize: 14,
+            marginBottom: 20,
+          }}
+        >
+          {data.title}
+        </h3>
+        <Spacer />
+        <div style={{ float: 'right', display: 'flex' }}>
+          <TimeIcon style={{ marginTop: 3 }} color="green" />
+          <Text style={{ marginLeft: 10, fontSize: 14, fontWeight: 'bold' }} color="green">
+            {Number(data.totalTime) + Number(data.prepTime)} min
+          </Text>
+        </div>
+      </Flex>
+
+      {data.mainImage?.fluid == null ? (
```

Comment:
I would wrap both the placeholder and the actual image in one big `Box` - the image is `fluid` so it'll fill the `Box`, so we can make the placeholder fill the `Box` as well and they'll be the same size